docs/stringvalidator/format.md

```diff
@@ -0,0 +1,73 @@
+---
+hide:
+    - navigation
+---
+# `Format`
+
+This validator is a generic validator for checking if the string is a valid format.
```

Comment:
The Format validator is a flexible utility designed to validate whether a given string adheres to specific predefined formats. It supports multiple validation types, enabling you to ensure strings meet requirements such as Base64 encoding, UUIDs, etc